src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -63,6 +63,13 @@ public TestRunSummary(String id) {
     private String startUserJourneyTimestamp;
     private String endUserJourneyTimestamp;
     private String endTimestamp;
+    // Include the user-provided resourceFileName of test config in the summary:
+    //  perf/**.json, integration/**.json, resiliency/**.json
+    // This can be used to facilitate grouping of test runner results on the dashboard.
+    private String resourceFileName;
+    // Include user-provided TestSuite name in the summary:
+    // This can be used to facilitate grouping of test runner results on the dashboard.
+    private String testSuiteName;
```

Comment:
I think the `testSuiteName` should live outside of `TestConfiguration` since `TestSuite` literally is a container of `TestConfiguration`s even it may contain only a single configuration.